MSAL/test/unit/ios/unit-test-host/AppDelegate.h

```diff
@@ -0,0 +1,36 @@
+//------------------------------------------------------------------------------
```

Comment:
Logger in the test and automation app was causing exceptions to be thrown, it's just before when the code was being pulled in as a static lib it ran fast enough that we usually ran all of the tests and exited out before hitting the exception case, but with framework init time it slowed down just enough to hit the exception and fail.